etc/et.cfg

```diff
@@ -3,3 +3,6 @@
 
 [Networking]
 port = 2022
+
+[Debug]
+verbose = 0
```

Comment:
Since you have the context at the moment, can you also add a silent flag?  It can also be 0 by default, but with a value of 1 we do not log anything to file.  This can address #63

Reply:
To me it makes more sense that "silent" means we only do log(ERROR) and ignore all INFO and WARNING. Or do we want to disable all the logs?

Reply:
Never mind :) It definitely better to disable all logs in a "silent" mode.